bot/exts/moderation/slowmode.py

```diff
@@ -58,6 +58,10 @@ async def set_slowmode(self, ctx: Context, channel: Optional[TextChannel], delay
             log.info(f'{ctx.author} set the slowmode delay for #{channel} to {humanized_delay}.')
 
             await channel.edit(slowmode_delay=slowmode_delay)
+            if channel.id == Channels.python_general:
```

Comment:
Why are we tracking only one channel? I think having multiple samples would be worth it, like we often have #ot0 in slowmode.

Reply:
Maybe we should track all channels?

Reply:
That was the plan, but channels can change names (especially ot), while having it by id would be pretty annoying, and we don't really need other channels for what this is intended for, so I agreed with @jb3 to limit it to just pygen.

Reply:
I don't know how it looks on the moderation front for slowmode, but maybe discordpy would also be nice here. Dynamic tracking doesn't sound too good for channels that get a sm set rarely

Reply:
We could have a mapping of usually slowmode-ed channel IDs to human readable name, and use that before submitting the stats.

Reply:
I guess I could add dpy and ot0 in case it's needed in the future. It's not super important to me though.

Reply:
Sounds good